flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
wait what?

Reply:
It's because io.qt.QtCreator is x86 only and this extension targets it. I raised my concern about this approach above.

Reply:
Should not be an issue to open it up for other architectures.

Reply:
QtCreator being x86_64 only is another question I had elsewhere on a higher level.